lang/python/python-libselinux/Makefile

```diff
@@ -47,50 +49,30 @@ define Package/python-libselinux/description
 	This package provides the Python bindings for libselinux.
 endef
 
-# Needed to link libselinux utilities, which link against
-# libselinux.so, which indirectly depends on libpcre.so, installed in
-# $(STAGING_DIR_HOSTPKG).
-HOST_LDFLAGS += -Wl,-rpath="$(STAGING_DIR_HOSTPKG)/lib"
-
-HOST_MAKE_FLAGS += \
-	PREFIX=$(STAGING_DIR_HOSTPKG) \
-	SHLIBDIR=$(STAGING_DIR_HOSTPKG)/lib
-
-$(eval $(call HostBuild))
-
 MAKE_FLAGS += \
 	FTS_LDLIBS=-lfts \
 	SHLIBDIR=/usr/lib \
 	PYTHON=$(PYTHON3) \
 	PYINC="-I $(PYTHON3_INC_DIR)" \
 	PYLIBS="-lpython$(PYTHON3_VERSION)" \
+	$(PYTHON3_VARS)
 
-define Build/Compile/python-libselinux
+define Build/Compile
 	$(call Build/Compile/Default,swigify pywrap)
 endef
 
-define Build/Install/python-libselinux
+define Build/Install
 	$(call Build/Install/Default,install-pywrap)
 endef
 
-define Build/Compile
-	$(call Build/Compile/Default,all)
-	$(Build/Compile/python-libselinux)
-endef
-
-define Build/Install
-	$(call Build/Install/Default,install)
-	$(Build/Install/python-libselinux)
+define Build/InstallDev
+	$(INSTALL_DIR) $(1)$(PYTHON3_PKG_DIR)
+	$(CP) $(PKG_INSTALL_DIR)$(PYTHON3_PKG_DIR)/* $(1)$(PYTHON3_PKG_DIR)
 endef
 
-define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/* $(1)/usr/include/
-	$(INSTALL_DIR) $(1)/usr/lib/pkgconfig
-	$(INSTALL_DATA) $(PKG_INSTALL_DIR)/usr/lib/pkgconfig/libselinux.pc $(1)/usr/lib/pkgconfig/
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/* $(1)/usr/lib/
+define Package/python-libselinux/install
+	$(INSTALL_DIR) $(1)$(PYTHON3_PKG_DIR)
+	$(CP) $(PKG_INSTALL_DIR)$(PYTHON3_PKG_DIR)/* $(1)$(PYTHON3_PKG_DIR)
 endef
 
-$(eval $(call Py3Package,python-libselinux))
```

Comment:
As we are using custom `Build/Install` and `Build/Compile` there is no need to call `Py3Package`.

Reply:
Applied in last push.